src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -455,3 +464,112 @@ def resume_privacy_request(
     ).submit(from_webhook=webhook)
 
     return privacy_request
+
+
+def review_privacy_request(
+    db: Session, cache: FidesopsRedis, request_ids: List[str], process_request: Callable
+) -> BulkReviewResponse:
+    """Helper method shared between the approve and deny privacy request endpoints"""
+    succeeded: List[PrivacyRequest] = []
+    failed: [Dict[str, Any]] = []
```

Comment:
Should this be `List[Dict[str, Any]]` instead of `[Dict[str, Any]]`?

Reply:
whoops, thanks for catching!